src/components/FileUpload.tsx

```diff
@@ -122,67 +217,70 @@ export default function FileUpload() {
                             </Typography>
                         </div>
 
-                        {Object.keys(datasets).length > 0 && (
```

Comment:
Is there something to replace this? The user should be able to see all of their uploaded datasets in the datasets panel, unless we change how the user accesses datasets

Reply:
Never mind, see the comment on topview